src/quo2/components/list_items/account/component_spec.cljs

```diff
@@ -0,0 +1,114 @@
+(ns quo2.components.list-items.account.component-spec
+  (:require [test-helpers.component :as h]
+            [quo2.components.list-items.account.view :as account]
+            [quo2.foundations.colors :as colors]))
+
+(h/describe "List items: account"
+  (h/test "default render'"
+    (h/render [account/view])
+    (h/is-truthy (h/query-by-label-text :container)))
+
+  (h/test "on-press-in changes state to :pressed"
+    (h/render [account/view])
+    (h/fire-event :on-press-in (h/get-by-label-text :container))
+    (h/wait-for #(h/has-style (h/query-by-label-text :container)
+                              {:height            56
+                               :borderRadius      12
+                               :backgroundColor   (colors/custom-color :blue 50 5)
+                               :flexDirection     :row
+                               :alignItems        :center
+                               :paddingHorizontal 12
+                               :paddingVertical   6
+                               :justifyContent    :space-between})))
```

Comment:
Not sure if we get a value from this kind of `strict-styling` tests, given that styles might be changing and it would add extra work. Could instead use snapshot testing (_mehh_) which would have a similar result (but easier to maintain), or just do the check for the background color, since it seems like it's the only style that changes.

Reply:
Agree, see my comment in this other PR -> https://github.com/status-im/status-mobile/pull/17398#discussion_r1336542923

I'll update on this once I have more inputs either on this or the other PR

Reply:
For the beginning I would just strip the tests of all the unnecessary styles and just keep the style u test for (here it's `backgroundColor`). For me it would be enough to have it merged, and then decide on the usefulness of these tests in a follow-up. We already have quite a few tests that check for styles in this way, so I guess it should be fine.

Reply:
Nice, I thought `toHaveStyle` checked for the entire style map, but it works also checking for a specific style prop, so I simplified it as suggested :)